knative-serving/config-applications/startup-probes-for-serving.adoc

```diff
@@ -0,0 +1,21 @@
+:_content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="startup-probes-for-serving"]
+= Startup probes
+:context: startup-probes-for-serving
+
+toc::[]
+
+Startup probes verify whether a service has started successfully, helping to reduce cold start times for containers with slow startup processes.
+Startup probes run only during the container's initialization phase and do not execute periodically. If a startup probe fails, the container adheres to the defined `restartPolicy`.
```

Comment:
Does this have to be in backticks, or could it just be said in plain language?

```suggestion
Startup probes run only during the container's initialization phase and do not execute periodically. If a startup probe fails, the container adheres to the defined restart policy.
```

Reply:
I think we can keep this because it's a defined variable in the config file. However, if it would be better not to have backticks in the assembly, I am open to changing it.